### DISCUSSION TITLE
1.3.0-beta.1 发布之后，文档站点无法访问了

### DISCUSSION THREAD

**Discussion by ofyann** [element-plus/element-plus#5077] [Q&A] [answered]
https://element-plus.org/zh-CN/ 
![image](https://user-images.githubusercontent.com/32095100/147806205-b847a847-347f-4773-b901-1855e9978666.png)
有一个 js 文件 404 了，访问直接白屏。

**Comment by sxzz** [comment, marked as answer]
临时可以使用 https://staging.element-plus.org/

**Reply by ofyann** [reply]
👌

**Reply by ofyann** [reply]
修复完了，gitee 版本会同步改吗。刚给人家看发现还是旧版本的~

**Reply by sxzz** [reply]
Fixed in #5081

**Reply by sxzz** [reply]
> 修复完了，gitee 版本会同步改吗。刚给人家看发现还是旧版本的~


Updated.

**Reply by ofyann** [reply]
谢谢~ 👻
